collector/utils.go

```diff
@@ -9,7 +9,10 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-const catalogProto = "catalog://"
+const (
+	catalogProto = "catalog://"
+	searchLimit  = "1000"
```

Comment:
Any reason for 1000 ?

Reply:
I guessed that `1000` was the maximum on Rancher api, but i saw that this is not true from rancher v2.3.0, https://github.com/rancher/rancher/commit/f2b601cc6f4d13674b998a6a24f09efc26629d8e. Updating to `searchLimit = -1`